apps/studio/src/components/TabQueryEditor.vue

```diff
@@ -479,43 +493,6 @@
       result() {
         return this.results[this.selectedResult]
       },
-      individualQueries() {
```

Comment:
@not-night-but I removed a bunch of queries related computations because it's being done inside the SQL Text Editor now. I feel like this shouldn't affect the new JavaScript Text Editor right?